### ISSUE TITLE
Subobjects are not sortable

### ISSUE THREAD

**Issue by DominicBoettger** [Zizzamia/ng-tasty#140]
It is not possible to sort field which are in a sub JSON - structure.

Example:
```js
  'header': [
            {
              'key': 'myfield',
              'name': 'MyFIELD'
            },
            {
              'key': 'subjobject.name',
              'name': 'subobject name'
            }]
```

Click on myfield sorts it correctly.
Click on subobject.name only works once. 

Best regards
Dominic

**Comment by macem** [comment]
I created fix for this, please check https://github.com/macem/ng-tasty-fix/commit/13756fc4fb88ca76f61dd849ad41b34053591ae9

**Comment by Zizzamia** [comment]
ThankYou to help me fix this issue. Can you please add test case for this issue. @macem

**Comment by patrickchampclos** [comment]
Here is a plunker to illustrate this issue with your website exemple: http://plnkr.co/edit/pLSwYOmUgttgy3MWWMEZ

I just relocate "star" and "sf-Location" keys in a metadata object and I link "metadata.star" and "metadata.sf-Location" in headers key.

**Comment by Zizzamia** [comment]
@macem can you make a PR with test please.

**Comment by Zizzamia** [comment]
Happy new Year ✨ just released the latest version `v0.6.1` of ngTasty
